leoism/step/portfolio/src/main/java/com/google/sps/data/CommentInformation.java

```diff
@@ -22,19 +22,26 @@
   private final long timestamp;
   private final double sentimentScore;
 
-  public CommentInformation(String comment, String name, long timestamp,
-                            double sentimentScore) {
+  public CommentInformation(String comment, String name, long timestamp, double sentimentScore) {
```

Comment:
The class comment needs a little more information, also it isn't obvious what kind of timestamp is stored, if it is a unix timestamp, please specify in the class comment and the timestamp getter.

Reply:
Done